packages/hardhat/test/e2e/crab-Migration.ts

```diff
@@ -59,25 +76,51 @@ describe("Crab Migration", function () {
         dToken = await ethers.getContractAt("IDToken", dTokenAddress);
         eulerExec = await ethers.getContractAt("IEulerExec", eulerExecAddress);
         crabStrategyV1 = await ethers.getContractAt("CrabStrategy", crabV1Address);
+        controller = await ethers.getContractAt("Controller", squeethControllerAddress);
         oracle = await ethers.getContractAt("Oracle", oracleAddress);
 
         deposit1Amount = await crabStrategyV1.balanceOf(crabV1Whale);
         deposit2Amount = await crabStrategyV1.balanceOf(crabV1Whale2);
+        deposit3Amount = await crabStrategyV1.balanceOf(crabV1Whale3);
+        deposit4Amount = await crabStrategyV1.balanceOf(crabV1Whale4);
+        deposit5Amount = await crabStrategyV1.balanceOf(crabV1Whale5);
 
         // Send Crab shares to d1
         await provider.send('hardhat_impersonateAccount', [crabV1Whale]);
-        const signer1 = await ethers.provider.getSigner(crabV1Whale);
+        const signer1 = ethers.provider.getSigner(crabV1Whale);
         await crabStrategyV1.connect(signer1).transfer(d1.address, deposit1Amount);
         await provider.send('evm_mine', []);
         await provider.send('hardhat_stopImpersonatingAccount', [crabV1Whale]);
 
         // Send Crab shares to d2
         await provider.send('hardhat_impersonateAccount', [crabV1Whale2]);
-        const signer2 = await ethers.provider.getSigner(crabV1Whale2);
+        const signer2 = ethers.provider.getSigner(crabV1Whale2);
         await crabStrategyV1.connect(signer2).transfer(d2.address, deposit2Amount);
         await provider.send('evm_mine', []);
         await provider.send('hardhat_stopImpersonatingAccount', [crabV1Whale2]);
 
+        // Send crab shares to d3
+        await provider.send('hardhat_impersonateAccount', [crabV1Whale3]);
+        const signer3 = ethers.provider.getSigner(crabV1Whale3);
+        await crabStrategyV1.connect(signer3).transfer(d3.address, deposit3Amount);
+        await provider.send('evm_mine', []);
+        await provider.send('hardhat_stopImpersonatingAccount', [crabV1Whale3]);
+
+        // Send crab shares to d4
+        await owner.sendTransaction({ to: crabV1Whale4, value: ethers.utils.parseEther('.1') })
```

Comment:
why giving whale 4 and 5 extra eth?

Reply:
They don't have enough ETH to pay gas lol

Reply:
stacking too much eth in the vault and not enough in their wallet